runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -494,4 +558,30 @@ private OverlayBounds buildSnapCorners()
 			new Rectangle(rightChatboxPoint, SNAP_CORNER_SIZE),
 			new Rectangle(canvasTopRightPoint, SNAP_CORNER_SIZE));
 	}
+
+	private MenuEntry[] createRightClickMenuEntries(Overlay overlay)
+	{
+		List<MenuOption> options = overlay.getMenuOptions();
+		List<MenuEntry> entries = new ArrayList<>();
+
+		int optionIndex = 0;
+		for (MenuOption option : options)
+		{
+			String name = overlay.getOverlayName() != null ? overlay.getOverlayName() : overlay.getName();
+			final MenuEntry entry = new MenuEntry();
+			entry.setOption(option.getOptionText());
+			entry.setTarget(ColorUtil.colorTag(ColorScheme.BRAND_ORANGE) + name);
```

Comment:
Is there any reason for using BRAND_ORANGE rather than the default menu target color?

Reply:
Didn't know it existed honestly, i'll change it.